### DISCUSSION TITLE
Recommendations for tablet with Linux?

### DISCUSSION THREAD

**Discussion by harpone** [xournalpp/xournalpp#1742] [General]
I've usually had an Android tablet (Galaxy Note, Note Pro etc) but recently got Pixel Slate. I usually used LectureNotes for written notes (math, illustrations), but that doesn't work too well on the slate. Also OneNote is a bit buggy and quid doesn't have the features I want.

Pixel Slate has now a Linux VM but the stylus doesn't work correctly yet (see #1444).

I was thinking of maybe switching to a tablet with a Linux distro (preferably Ubuntu). 

I'd appreciate some tips on HW + SW setups and use cases for Xournal++ note taking. Thanks!

**Comment by masac** [comment]
I use Wacom Bamboo Splash Pen Tablet (https://www.amazon.com/Wacom-Bamboo-Splash-Tablet-CTL471/dp/B0089VGPII) and Xournal++ on Linux Mint 19.3.

**Comment by cbm755** [comment]
I use an older 'Thinkpad 10' tablet bought in 2015.  Linux kernel support has been readily improving.  Last checked carefully with 5.3 in Fedora 32 where I think suspend was still a bit crashing.

**Comment by harpone** [comment]
Ok thanks for the suggestions! I'd definitely need a standalone device, so no Wacom Bamboo stuff unfortunately... 

TP tablet 10 seems nice! I could use something a bit bigger preferably (12") but still light weight.

I was looking at Surface Pro series... can anyone confirm that the pen is working OK on Linux?

**Comment by nicegamer7** [comment]
I'd look at

https://github.com/linux-surface/linux-surface/wiki/Supported-Devices-and-Features#feature-matrix

**Comment by MadsDyrmann** [comment]
I have a Dell Inspiron 7586 with stylus, which works great with Xournal++

**Comment by LittleHuba** [comment]
The Yoga series by Lenovo is a well supported series. The only thing I know of that doesn't work is the fingerprint sensor of the newer generations. A Google search will give you tons of resources what exactly is supported.

**Comment by Tall-Lobster** [comment]
@MadsDyrmann 

I have the same system but I am facing issues having xournal++ detecting the pen as it thinks it is a mouse.

if you could let me know if under Edit -> Preferences -> Input System if you get the stylus appearing and touchscreen other than the mouse.

**Comment by 42FordPrefect** [comment]
I've also got an old Thinkpad 10, running Ubuntu 19.10. While it was a lot of work, using a old linuxium iso and then updating it with do-release-upgrade, it's running a bit laggy (with GNOME) but quite stable.

**Comment by michaelcadilhac** [comment]
@harpone Did you go for one of the suggestions here?  My Galaxy Note Pro 12.2 with LectureNotes is constantly rebooting, battery swap didn't help, so I'm on the market for a 12.2-15 inch light device for note taking.  No Android tablets seems to hit the marks, so I'm looking at Xournal++ and 2-in-1 laptops or the Surface.  I'd appreciate your feedback!

**Comment by harpone** [comment]
@michaelcadilhac no didn't really find any of the suggestions interesting enough... I have a Pixel Slate and I'm just working with that for now.

**Comment by espoelstra** [comment]
If you aren't scared of a little work you might also check out Crouton and try running Xournal from there rather than the Crostini container. This would give more direct access to the hardware so you could potentially have the stylus be detected and change its behavior.

The downside of Crouton is it requires Developer Mode which wipes your system (basically removing anything in Downloads or other folders and in Crostini that you haven't backed up to Google Drive) and this has some security implications because Developer Mode lets you poke more deeply at the operating system, but worst  case scenario you should always be able to Power Wash or do a full USB Recovery if something goes really badly.

**Comment by michaelcadilhac** [comment]
@espoelstra Cheers!  Well, I do have crouton installed, but even with the touch target, there's no separate device for touchscreen and mouse (let alone stylus).  In its current state, this is _less_ useful than crostini.

**Comment by mizhozan** [comment]
[Weylus](https://github.com/H-M-H/Weylus) might also make your regular PC setup just like a linux tablet. There is some latency problem at least in my case but generally it convert a cheap tablet+stylus solution to a cintiq level graphic tablet. At least it's the cheapest solution.
